build.gradle

```diff
@@ -19,15 +19,19 @@ repositories {
 
 ext {
     kotlinVersion = '1.3.72'
-    springBootVersion = '2.1.5.RELEASE'
+    springBootVersion = '2.2.5.RELEASE'
     swaggerVersion = '2.9.2'
     feignVersion = '2.1.3.RELEASE'
 }
 
 dependencies {
-    implementation "org.springframework.boot:spring-boot-starter-web"
-    implementation "org.springframework.boot:spring-boot-starter-data-jpa"
-    implementation "org.springframework.boot:spring-boot-starter-security"
+    //Spring
+    implementation("org.springframework.boot:spring-boot-starter-security")
+    implementation("org.springframework.boot:spring-boot-starter-webflux")
+    implementation("org.springframework.boot:spring-boot-starter-data-mongodb-reactive")
+    implementation("org.springframework.boot:spring-boot-starter-data-redis-reactive")
+
+    compile group: 'javax.servlet', name: 'javax.servlet-api', version: '3.0.1'
```

Comment:
`compile` should be changed to `implementation`. 

Where are the braces?! Follow the same format for all of the dependencies.